README

```diff
@@ -12,4 +12,17 @@ autoreconf -vfi
 
 ./configure && make && make install
 
+Building enet -use vcpkg
+
+You can download and install enet using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
+   
+    git clone https://github.com/Microsoft/vcpkg.git
+    cd vcpkg
+    ./bootstrap-vcpkg.sh
+    ./vcpkg integrate install
+    vcpkg install enet
```

Comment:
The existing command line above is *not* indented. This file currently isn't real markdown.
I feel like it should be renamed to README.md and the other issues should be fixed first, OR, you should keep changes to plain text.